tests/build/build.go

```diff
@@ -637,7 +658,59 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				It(fmt.Sprintf("the PipelineRun should eventually finish successfully for component %s", componentName), func() {
 					Expect(f.AsKubeAdmin.HasController.WaitForComponentPipelineToBeFinished(component, "", 2, f.AsKubeAdmin.TektonController)).To(Succeed())
 				})
+
+				It("merging the PR should be successful", func() {
+					Eventually(func() error {
+						mergeResult, err = f.AsKubeAdmin.CommonController.Github.MergePullRequest(multiComponentGitSourceRepoName, prNumber)
+						return err
+					}, time.Minute).Should(BeNil(), fmt.Sprintf("error when merging PaC pull request #%d in repo %s", prNumber, multiComponentGitSourceRepoName))
```

Comment:
print error message in the Eventually output

Reply:
I think printing the error in `Eventually` func is automatically handled by gomega